.github/ISSUE_TEMPLATE/1-bug.yml

```diff
@@ -41,9 +41,9 @@ body:
         Link to repository or sandbox with runnable example of the issue.
 
         Some CodeSandbox starters:
-        - [remark only](https://codesandbox.io/s/remark-debug-ikwvx) (for markdown to markdown)
-        - [remark and rehype](https://codesandbox.io/s/remark-rehype-debug-4cz8v) (for markdown to html)
-        - [react-markdown](https://codesandbox.io/s/react-markdown-debug-9n4eg)
+        - [remark only](https://codesandbox.io/s/github/remarkjs/.github/tree/main/sandbox-templates/remark-with-parcel) (for markdown to markdown)
+        - [remark and rehype](https://codesandbox.io/s/github/remarkjs/.github/tree/main/sandbox-templates/remark-rehype-with-parcel) (for markdown to html)
+        - [react-markdown](https://codesandbox.io/s/github/remarkjs/.github/tree/main/sandbox-templates/react-markdown-with-create-react-app)
```

Comment:
temporary links until merged

* https://codesandbox.io/s/github/ChristianMurphy/.github-remark/tree/sandbox-starters-in-repo/sandbox-templates/remark-with-parcel
* https://codesandbox.io/s/github/ChristianMurphy/.github-remark/tree/sandbox-starters-in-repo/sandbox-templates/remark-rehype-with-parcel
* https://codesandbox.io/s/github/ChristianMurphy/.github-remark/tree/sandbox-starters-in-repo/sandbox-templates/react-markdown-with-create-react-app

optionally we could also link to stackblitz
* https://stackblitz.com/github/ChristianMurphy/.github-remark/tree/sandbox-starters-in-repo/sandbox-templates/remark-with-parcel
* https://stackblitz.com/github/ChristianMurphy/.github-remark/tree/sandbox-starters-in-repo/sandbox-templates/remark-rehype-with-parcel
* https://stackblitz.com/github/ChristianMurphy/.github-remark/tree/sandbox-starters-in-repo/sandbox-templates/react-markdown-with-create-react-app